tests/integration/apigateway/test_apigateway_import.py

```diff
@@ -722,3 +722,23 @@ def test_import_with_circular_models_and_request_validation(
         request = requests.post(url, json=wrong_request_schema)
         assert request.status_code == 400
         assert request.json().get("message") == "Invalid request body"
+
+    def test_import_with_stage_variables(self, import_apigw, aws_client, echo_http_server):
```

Comment:
nit: maybe we could just run the test against AWS using the `echo_http_server_post` fixture which uses `httpbin.org` when running against AWS and the default pytest fixture against LocalStack? Just so that the test can marked as `aws_validated`? We would just need to change the method to `POST`.

Reply:
Yep makes sense, let me fix that